packages/graph/hash_graph/lib/graph/src/types/schema/entity_type.rs

```diff
@@ -171,20 +176,25 @@ mod tests {
     }
 
     fn test_property_refs(entity_type: &EntityType, uris: impl IntoIterator<Item = &'static str>) {
-        let expected_property_references = uris.into_iter().map(Uri::new).collect::<HashSet<_>>();
+        let expected_property_references = uris
+            .into_iter()
+            .map(ToString::to_string)
+            .collect::<HashSet<_>>();
 
         let property_references = entity_type
             .property_type_references()
             .into_iter()
-            .map(PropertyTypeReference::reference)
-            .cloned()
+            .map(|reference| reference.uri().base_uri().to_string())
```

Comment:
Even if the idea lands well, let's not do it as part of this PR so we can get things moving.

I think we should be looking at code like this and trying to improve readability and convey intentions more. In my head something like this would be better:

`PropertyName::from(PropertyReference)`. 

I assume "expected_property_references" is actually checking the `requirements` or propertyNames in the properties blob, _rather_ than actually checking the "references".

Generally speaking encapsulating something like `PropertyName::from(PropertyReference)` also is encoding our expectations that property names are derived from the reference inside them.

Reply:
Generally a good idea. We could have `struct PropertyName(BaseURI)` or something similar for this.